app/Models/UserQuiz.php

```diff
@@ -50,6 +50,11 @@ public function isClosed(): Attribute
         return Attribute::get(fn(): bool => $this->closed_at <= Carbon::now());
     }
 
+    public function wasClosedManually(): bool
+    {
+        return $this->closed_at->ne($this->quiz->closeAt);
```

Comment:
Is it supposed to be like this?

Reply:
yes 
![image](https://github.com/user-attachments/assets/638d919d-cffb-4a5b-8a9e-6585d6360843)